prog/dftb+/lib_dftb/blockpothelper.F90

```diff
@@ -0,0 +1,77 @@
+!--------------------------------------------------------------------------------------------------!
+!  DFTB+: general package for performing fast atomistic simulations                                !
+!  Copyright (C) 2006 - 2020  DFTB+ developers group                                               !
+!                                                                                                  !
+!  See the LICENSE file for terms of usage and distribution.                                       !
+!--------------------------------------------------------------------------------------------------!
+
+#:include 'common.fypp'
+
+!> Helper routines for block charges (used in on-site corrections and DFTB+U)
+module dftbp_blockpothelper
+  use dftbp_accuracy, only : dp
+  use dftbp_commontypes, only : TOrbitals
+  implicit none
+
+  private
+  public :: AppendBlock_reduce
+
+contains
+
+  !> Adds block (dual) charges onto end of a 1D vector of reduced charges
+  subroutine AppendBlock_reduce(input, equiv, orb, output, isSkew)
```

Comment:
Could we use the occasion of touching this, to align the subroutine name with the style guide? E.g. `appendBlockReduced` or similar?